src/pages/NftInfluencersTrx/Table/Row/Info/Info.js

```diff
@@ -0,0 +1,108 @@
+import React, { useRef } from 'react'
+import cx from 'classnames'
+import { copy } from 'webkit/utils'
+import Svg from 'webkit/ui/Svg/react'
+import {
+  Activity,
+  getTwitterAccount,
+  Marketplace,
+} from '../../../../Index/Section/NftInfluencers/utils'
+import { dateDifferenceInWords } from '../../../../../utils/dates'
+import { capitalizeStr } from '../../../../../utils/utils'
+import styles from './Info.module.scss'
+
+const Info = ({ data, isOpened }) => {
+  const account = getTwitterAccount(data)
+  const { nft, datetime, trxHash, amount, currencyProject, marketplace } = data
+
+  const clearTimerRef = useRef()
+
+  function onCopyAddressClick(address) {
+    if (clearTimerRef.current) clearTimerRef.current()
+
+    const node = document.getElementById(address)
```

Comment:
Elements' `id` is not needed since the data can be accessed using `e.currentTarget.textContent`